features/_your_details/your_details_displaying.feature

```diff
@@ -3,8 +3,19 @@ Feature: Your Details displaying
   As a user of the WPCC tool,
   I want to see my details that I filled in on step one.
 
-Scenario: Display input details on later steps
-  Given I am on the Your Details step
-  When I fill in my details
+Scenario Outline: Display input details on later steps
+  Given that I am on the WPCC homepage in my own "<language>"
+  When I enter my age as "<age>"
+  And I select my gender as "<gender>"
+  And I select my salary frequency as "<salary_frequency>"
+  And I choose my contribution preference as "<contribution_preference>"
+  And I enter my salary as "<salary>"
   And I proceed to the next step
-  Then I should see my details summarised
+  Then I should see "<my_details>" summarised
+
+  Examples:
+      | language | age | gender    | salary_frequency | salary | contribution_preference | my_details                                                               |
+      | English  | 23  | Male      | per Year         | 6000   | Minimum                 | 23 years, male, £6,000 per year, part salary                             |
+      | Welsh    | 34  | Benywaidd | fesul 4 wythnos  | 36000  | Minimum                 | 34 blynyddoedd, benywaidd, £36,000 fesul 4 wythnos, nghyflog rhan Golygu |
+      | English  | 43  | Female    | per Month        | 6000   | Full                    | 43 years, female, £6,000 per month, full salary                          |
+      | Welsh    | 52  | Benywaidd | y Wythnos        | 36000  | Full                    | 52 blynyddoedd, benywaidd, £36,000 y wythnos, nghyflog llawn             |
```

Comment:
The thing I am battling with to decide the best approach to this, is how necessary all these examples are.. I feel like because the presenter and controller just return a string with the values. the tests to make sure different inputs produce different outputs should be covered within the unit tests and not here..
To me we probably only need one english example and one welsh example